docs/releases/g_1_1.md

```diff
@@ -13,5 +13,5 @@ This is the first production Create® 3 release.
 This release provides the robot's sensors and controls via topics, actions, and services, see [ROS 2 API](../../api/ros2/).
 
 The robot runs a webserver for updating software, configuring to the network, getting error logs from the robot, and configuring ROS 2 environment variables and parameters.
-This release has some known issues using the network configuration webserver page to associate with networks with non standard characters.
+This release has some known issues using the network configuration webserver page to associate with networks with alphanumeric 7-bit ASCII characters.
```

Comment:
the opposite -- if there are non-alphanumeric 7-bit ascii characters (IE, Unicode or `-` or ` ` or other such things) it can barf

Reply:
Fixed in fb02907